python/dgl/nn/pytorch/explain/gnnexplainer.py

```diff
@@ -35,28 +35,41 @@ class GNNExplainer(nn.Module):
         The learning rate to use, default to 0.01.
     num_epochs : int, optional
         The number of epochs to train.
+    edge_size : float, optional
+        A higher value will make the explanation edge masks more sparse by decreasing
+        the sum of the edge mask.
+    edge_ent : float, optional
+        A higher value will make the explanation edge masks more sparse by decreasing
+        the entropy of the edge mask.
+    node_feat_size : float, optional
+        A higher value will make the explanation node feature masks more sparse by
+        decreasing the mean of the node feature mask.
+    node_feat_ent : float, optional
+        A higher value will make the explanation node feature masks more sparse by
+        decreasing the entropy of the node feature mask.
     log : bool, optional
         If True, it will log the computation process, default to True.
     """
 
-    coeffs = {
-        'edge_size': 0.005,
-        'edge_ent': 1.0,
-        'node_feat_size': 1.0,
-        'node_feat_ent': 0.1
-    }
-
     def __init__(self,
                  model,
                  num_hops,
                  lr=0.01,
                  num_epochs=100,
+                 edge_size=0.005,
+                 edge_ent=1.0,
+                 node_feat_size=1.0,
+                 node_feat_ent=0.1,
```

Comment:
I suggest making some of those arguments key-words only. E.g.

```python
    def __init__(self,
                 model,
                 num_hops,
                 lr=0.01,
                 num_epochs=100,
                 *,
                 edge_size=0.005,
                 edge_ent=1.0,
                 node_feat_size=1.0,
                 node_feat_ent=0.1,
```

Reply:
I'm fine with the proposal.

Reply:
Updated